docs/MAINTAINERS-guidelines.md

```diff
@@ -0,0 +1,51 @@
+# Introduction
+All Hyperledger projects MUST have a `MAINTAINERS.md` file at the top-level directory of the source code. This document will provide guidelines on what should be included in the `MAINTAINERS.md` file.
+
+# Guidelines
+## List of Project Maintainers
+The first thing that MUST be included in the `MAINTAINERS.md` file is a list of the project's maintainers, both active and emeritus.
+
+It is recommended that the lists be sorted alphabetically and contain the maintainers name, GitHub ID, LFID, Company Affiliation, and Scope. The following shows the suggested format for the information:
+
+**Active Maintainers**
```

Comment:
Would adding the rocket chat id be useful for the active maintainers?

Reply:
I agree with Swetha, I find it difficult to contact people when all I have is their github id. I would favor adding both the RC id and an email address, with a requirement to have at least one or the other provided.

Reply:
Isn’t your LFID the same as your Rocket.Chat ID?

Reply:
Ah, good point. I guess it is. Maybe we could highlight that in having both in the header for dummies like me who don't think about it. :-)

Reply:
Changes made to reflect. See if this meets the needs.

Reply:
My rocket chat id differs from my LFID, though I use my LFID to sign into rocket chat.

Reply:
Thanks, @swetharepakula. I updated.